prometheus.yml

```
@@ -54,8 +54,8 @@ scrape_configs:
 
   - job_name: 'kafka-exporter'
     static_configs:
-        - targets: [ 'kafka-exporter:9308' ]
+      - targets: [ 'kafka-exporter:9308' ]
 
   - job_name: 'redis-exporter'
     static_configs:
-        - targets: [ 'redis-exporter:9121' ]
+      - targets: [ 'redis-exporter:9121' ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)